pwndbg/heap/ptmalloc.py

```diff
@@ -5,17 +5,217 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+from collections import OrderedDict
+
 import gdb
 
 import pwndbg.events
 import pwndbg.typeinfo
+from pwndbg.constants.ptmalloc import *
+
+
+class Heap(pwndbg.heap.heap.Heap):
+
+    def __init__(self):
+        # Global ptmalloc objects
+        self._main_arena    = None
+        self._mp            = None
+
+        # Symbols and types
+        self._malloc_chunk  = None
+        self._malloc_state  = None
+        self._mallinfo      = None
+        self._malloc_par    = None
+
+    @property
+    def main_arena(self):
+        from pwndbg.color import bold
+        from pwndbg.color import red
+
+        if not self._main_arena:
+            main_arena_symbol = gdb.lookup_symbol('main_arena')[0]
```

Comment:
I've just added code to `master` which uses `lookup_symbol` in `pwndbg.symbol.address`